core/src/main/resources/hudson/model/Node/help-numExecutors_ru.html

```diff
@@ -1,15 +1,17 @@
 ﻿<div>
   Эта опция контролирует количество одновременных сборок, которые может производить Jenkins.
-  Таким образом, значение определяет результирующую нагрузку на сервер, которую может 
-  вызвать. Для начала установите это значение равным количеству процессоров в вашей системе.
-
   <p>
-  Увеличивая это значение далее, вы можете заметить увеличение продолжительности каждой 
-  сборки, однако результирующая пропускная способность увеличитcя, так как это позволит 
-  процессору выполнять одну из сборок в то время как другая сборка ожидает 
-  окончания операции ввода-вывода.
-
+    Таким образом, значение определяет результирующую нагрузку на сервер, которую может
+    вызвать. Для начала установите это значение равным количеству процессоров в вашей системе.
+    Увеличивая это значение далее, вы можете заметить увеличение продолжительности каждой
+    сборки, однако результирующая пропускная способность увеличитcя, так как это позволит
+    процессору выполнять одну из сборок в то время как другая сборка ожидает
+    окончания операции ввода-вывода.
   <p>
-  Используя Jenkins в режиме мастер/рабочий, установка значения в 0 запретит мастеру 
-  выполнять сборки самостоятельно.
+    Узлы должны иметь минимум один сборщик. Чтобы временно приостановить выполнение сборок
+    на узле, используйте кнопку <i>Пометить данный узел как временно недоступный</i> на
+    странице настройки узла.
+  <p>
+    Это не применимо для мастер-узла &mdash; установка значения в 0 запретит мастеру
```

Comment:
This phrase needs some clarification in English as well. Flyweight tasks can be executed even if the number of executors is zero